src/AutorestSwift/JsonRpc/AutorestPlugin.swift

```diff
@@ -98,22 +93,13 @@ class AutorestPlugin {
                 .success(.list([.string("swift")]))
             )
         case "process":
-            guard let sessionId = params.asList?.last?.asString else {
-                logger.log("Process request get invalid session id")
-                callback(.failure(RPCError(
-                    kind: .invalidParams,
-                    description: "Process request get invalid session id"
-                )))
-                return
-            }
-
-            plugin.sessionId = sessionId
+            plugin.sessionId = params.asList?.last?.asString
             startChannelClient(context: context)
         // TODO: Sending back Process response will stop Autorest.
         // callback(.success(.bool(true)))
         default:
             callback(.failure(RPCError(kind: .invalidMethod, description: "Incoming Handler get invalid method")))
-            logger.log("invalid method: \(method)")
+            FileLogger.instance.log("invalid method: \(method)")
```

Comment:
Similarly here. If the method is invalid, we should fatalError the process. This is basically true of any failure in the case of this particular tool.

Reply:
Fixed